src/pages/Governance/Proposal/card/ConfirmationModal.tsx

```diff
@@ -0,0 +1,173 @@
+import React from "react";
+import {
+  Typography,
+  Modal,
+  Box,
+  Divider,
+  Button,
+  Stack,
+  IconButton,
+} from "@mui/material";
+import {grey} from "../../../../themes/colors/aptosColorPalette";
+import {useTheme} from "@mui/material/styles";
+import {Theme} from "@mui/material/styles";
+import {
+  voteFor,
+  voteAgainst,
+  primaryColor,
+  negativeColor,
+  primaryColorOnHover,
+  negativeColorOnHover,
+  primaryColorWithOpacity,
+  negativeColorWithOpacity,
+} from "../../constants";
+import CheckCircleOutlinedIcon from "@mui/icons-material/CheckCircleOutlined";
+import CloseOutlinedIcon from "@mui/icons-material/CloseOutlined";
+import CancelOutlinedIcon from "@mui/icons-material/CancelOutlined";
+
+interface CardBoxProps {
+  theme: Theme;
+  children?: React.ReactNode;
+}
+
+const CardBox = ({theme, children}: CardBoxProps) => {
+  return (
+    <Box
+      sx={{
+        position: "absolute" as "absolute",
```

Comment:
What is the purpose of `as "absolute"`?

Reply:
Copied it from MUI demo code. I don't think it's needed here. Removing. Thanks for catching it.